### ISSUE TITLE
Duplicate in .ssh/config when using ```coder config-ssh```

### ISSUE THREAD

**Issue by cfouche3005** [coder/coder#4258]
![image](https://user-images.githubusercontent.com/51720655/192987893-9056741f-2627-44fc-ae53-06f451d3f8f2.png)

When using the ```coder config-ssh``` , It create two entries in Visual Studio Code, one ```code.workspacename``` and one ```code.workspacename.main```. Is it an expected behavior or is it a bug ? Can we disable it ?

**Comment by matifali** [comment]
there should be a way to disable it if we are not using multiple `coder_agents`. for one `coder_agent` they are technically identical.

**Comment by matifali** [comment]
What is the purpose for these duplicate entries?

**Comment by kylecarbs** [comment]
If we're not using multiple agents, I suppose the expected behavior would be to use the shortened-form. @mafredri I'm curious for your thoughts. If you agree, I'll happily make the change!

**Comment by matifali** [comment]
Yes, all example templates and even the dogfood template have a single agent. I do to know any use cases where multiple agents can be helpful.

**Comment by kylecarbs** [comment]
Multiple agents can be used when multiple services are provisioned as part of a single development environment. Our telemetry shows that this use-case is valid.

**Comment by mafredri** [comment]
I agree with Kyle, if anything needs to change, I'd prefer to remove the shortcut. In those situations the user could still define the shortcut themselves via:

```
Host coder.*
    HostName %h.main
```

Assuming the default agent is called `main`.

**Comment by matifali** [comment]
Ok. So may something in coder config-ssh check to see if there is only one agent and then don't add the extra entry.

**Comment by kylecarbs** [comment]
Yup!

**Comment by mafredri** [comment]
> Ok. So may be something in `coder config-ssh` to check if there is only one agent and then don't add the extra entry.

We used to do this, but it was changed for the sake of consistency. If the name sometimes has the extra suffix, and other times not, that might be confusing to our users as well. Granted, multiple agents is a less common use-case.

It’s unclear to me what the best approach here is, but one of:
1. Add comment explaining why there are two
2. Remove one of the two (which?)
3. Add an option `--use-short-name` (or long) or something similar
